models/nxos/ospf_global/nxos_ospf_global.yaml

```diff
@@ -0,0 +1,529 @@
+---
+GENERATOR_VERSION: '1.0'
+
+ANSIBLE_METADATA: |
+    {
+        'metadata_version': '1.1',
+        'status': ['preview'],
+        'supported_by': 'network'
+    }
+
+NETWORK_OS: nxos
+RESOURCE: ospf_global
+COPYRIGHT: Copyright 2020 Red Hat
+
+DOCUMENTATION: |
+  module: nxos_ospf_global
+  version_added: 2.10
+  short_description: Manages OSPF configuration on devices running Cisco NX-OS.
+  description: 
+    - This module global OSPF configuration on devices running Cisco NX-OS.
+  author: Nilashish Chakraborty (@NilashishC)
+  options:
+    config:
+      description: A list of OSPF process configuration.
+      type: list
+      elements: dict
+      suboptions:
+        afi:
+          description:
+            - The Address Family Indicator.
+          type: str
+        instances:
```

Comment:
@GomathiselviS in NX-OS we can have multiple OSPF processes running, each identified by its process tag/id. So, we'd have to keep this as a list of dictionaries, instead of a dictionary for proper facts gathering.